### DISCUSSION TITLE
QSlider - v-slot:marker-label-group="{ markerList }"

### DISCUSSION THREAD

**Discussion by saniko** [quasarframework/quasar#12975] [General - Components / Directives / etc]
Hello,
I'm using the V1 example for marker-labels slots:
https://v1.quasar.dev/vue-components/slider#marker-labels

The example shows an array being used "markerList" to render the individual marks
```
  <q-slider
      class="q-mt-xl"
      v-model="secondModel"
      color="orange"
      markers
      :min="0"
      :max="5"
      marker-labels
    >
      <template v-slot:marker-label-group="{ markerList }">
        <div
          v-for="val in 4"
          :key="val"
          class="cursor-pointer"
          :class="markerList[val].classes"
          :style="markerList[val].style"
          @click="secondModel = val"
        >{{ val }}</div>

        <q-icon
          v-for="val in [0, 5]"
          :key="val"
          :class="markerList[val].classes"
          :style="markerList[val].style"
          size="sm"
          color="orange"
          :name="val === 0 ? 'volume_off' : 'volume_up'"
          @click="secondModel = val"
        />
      </template>
    </q-slider>
```

Where can I find documentation regarding this array?
What is the exact CSS class I need to use?

Thanks

**Comment by saniko** [comment]
Anyone?

**Reply by pdanpdan** [reply]
In the documentation, in the slots tab of the api

**Reply by saniko** [reply]
Thanks,
I saw the documentation, I meant the actual CSS and styles.
if you need to render more than 4 options, the code fails.
I toke if from the dev tools, is it right?

`  const markerList = [
     {style: 'left: 0%;',   class:'cursor-pointer q-slider__marker-labels q-slider__marker-labels--h-standard q-slider__marker-labels--h-ltr'},
     {style: "left: 25%;",   class:'cursor-pointer q-slider__marker-labels q-slider__marker-labels--h-standard q-slider__marker-labels--h-ltr'},
     {style: "left: 50%;",class:'cursor-pointer q-slider__marker-labels q-slider__marker-labels--h-standard q-slider__marker-labels--h-ltr'},
     {style: "left: 75%;", class:'cursor-pointer q-slider__marker-labels q-slider__marker-labels--h-standard q-slider__marker-labels--h-ltr'},
     {style: "left: 100%;", class:'cursor-pointer q-slider__marker-labels q-slider__marker-labels--h-standard q-slider__marker-labels--h-ltr'}
   ]`

**Reply by pdanpdan** [reply]
Created here: https://github.com/quasarframework/quasar/blob/63d0260337e892ac7455f2b1bb1d4eb0ebe6bb95/ui/src/components/slider/use-slider.js#L295

markerLabelClass here: https://github.com/quasarframework/quasar/blob/63d0260337e892ac7455f2b1bb1d4eb0ebe6bb95/ui/src/components/slider/use-slider.js#L288

getMarkerLabelStyle here: https://github.com/quasarframework/quasar/blob/63d0260337e892ac7455f2b1bb1d4eb0ebe6bb95/ui/src/components/slider/use-slider.js#L362
